numpy/ma/tests/test_core.py

```diff
@@ -1361,19 +1361,18 @@ def test_eq_with_None(self):
         # With partial mask
         with suppress_warnings() as sup:
             sup.filter(FutureWarning, "Comparison to `None`")
```

Comment:
Was this still needed?

Reply:
Looks like it may not be now, could try removing it. Which reminds me that there are a lot of expired deprecations in 1.15 that we should deal with soon.